routers/staff.router.js

```diff
@@ -93,6 +93,17 @@ staffRouter.put('/:fireID/deactivate', async (req, res) => {
     }
 });
 
+//POST adding new youths
+staffRouter.post("/newYouth", async (req, res) => {
+    const newStaff = new Staff({...req.body})
```

Comment:
won't compile because Staff is not imported, move methods to controller, router calls method